crates/rust-analyzer/src/config.rs

```diff
@@ -1967,7 +1967,7 @@ fn get_field<T: DeserializeOwned>(
     alias: Option<&'static str>,
     default: &str,
 ) -> T {
-    // XXX: check alias first, to work-around the VS Code where it pre-fills the
+    // XXX: check alias first, to work around the VS Code where it pre-fills the
```

Comment:
Unlike an earlier case where `workaround` was used as a noun (where the `-` can be dropped), here `work around` is being used as a verb, and nouns tend to lose `-`s, verbs do not.